dmcontent/govuk_frontend.py

```diff
@@ -1,15 +1,55 @@
 """
 Create forms to answer questions using govuk-frontend macros.
 
-The main function in this module is `from_question`. It should be possible to
-do everything you might want to do just by calling `from_question` with a
-content loader Question.
 
-Read the docstring for `from_question` for more detail on how this works.
+This module aims to solve the developer need of
+
+    Given a content loader Question
+    I want to create a form element using the GOV.UK Design System
+    So that the user gets a good experience
+
+in a way that requires the developer to know as little as possible about the
+Question object in question.
+
+The main function in this module is `render_question()`. It will take a content
+loader Question and turn it into HTML that can be used directly in a template.
+
+`render_question()` needs to be called from within a Jinja2 template, so it
+should be added to your template environment. You can add it to all your
+templates by adding it to your environment globals like so:
+
+    >>> import jinja2
+    >>> from dmcontent.govuk_frontend import render_question
+    >>> env = jinja2.Environment()
+    >>> env.globals["render_question"] = render_question
+
+You will also need to make sure all the GOV.UK (and Digital Marketplace) macros
```

Comment:
Are these all required or could you just import, for example, govukInput if that's all you needed?

Reply:
You don't have to import them all, but you're in danger of getting a runtime error if you're not sure of what you're doing.

Reply:
Although now I think about it that's always true in Python/Jinja, calling a function that doesn't exist is a runtime error 😁